Card

camilagerarde opened the pull request:
- Refatoração da função showPokemons com inclusão de Template String para limpeza do código em main.js

- Inclusão de Types e Weaknesses no objeto infosCard, na função selectInfosToShow do data.js.

- Inclusão das informações Tipo, Fraquezas e Spawn no card apresentado na tela.

- Definição de estilo para os cards.

- Organização do css com os blocos de instrução em ordem alfabética.

camilagerarde committed:
Inclusão da tag div antes de section

camilagerarde committed:
Merge branch 'master' of https://github.com/naereloire/SAP004-data-lovers

camilagerarde committed:
style card

camilagerarde committed:
Inclusão de tipos e fraquezas no objeto infosCard

camilagerarde committed:
Inclusão da função showPokemon para o card na tela

camilagerarde committed:
Correção no nome da class

camilagerarde committed:
atualização da função no data de acordo com master

camilagerarde committed:
Alinhamento do código com a master

camilagerarde committed:
alinhamento do código com a master

camilagerarde committed:
funções alteradas

camilagerarde committed:
Alinhamento do código para pr

camilagerarde committed:
Alinhamento do código com a função showPokemons

camilagerarde committed:
Merge branch 'master' into card

naereloire commented on src/main.js:
```diff
@@ -7,36 +7,31 @@ let arrayPokemon = data.pokemon
-    let getDivCards = document.getElementById("local-cards")
-    getDivCards.innerHTML = ""
-
-    if (arrayPokemon.length == 0) {
```
Camis, é esse trecho aqui, que verifica se o resultado da busca foi vazio ou um elemento q não existe no array.
Acho que podemos manter, só adicionar ele no seu código dentro da showPokmons.

 `if (arrayPokemon.length == 0) {	
        getDivCards.innerHTML = "<br>Resultado não encontrado</br>"	
    }	
`

naereloire reviewed (changes requested):
Inclusão de trecho de verificação de array vazio ou inexistente.

camilagerarde committed:
Manter o if na função p "resultado não encontrado"

Mantive as linhas do código, para caso o usuário não coloque dados na busca apareça a mensagem "Resultado não encontrado".
Modificações feitas para funcionamento da função.
Alterado também a comparação de == para ===.

camilagerarde committed:
Alteração de Spawn para Chance de aparecer em card

camilagerarde reviewed (commented):
Mudanças feitas de acordo com solicitação
